examples/model_selection/plot_learning_curve.py

```diff
@@ -126,3 +126,128 @@ def plot_learning_curve(estimator, title, X, y, ylim=None, cv=None,
 plot_learning_curve(estimator, title, X, y, (0.7, 1.01), cv=cv, n_jobs=4)
 
 plt.show()
+
+###############################################################################
```

Comment:
Is there a way to actually integrate this into the example above, rather than having an entirely separate demonstration of this functionality?

Reply:
I guess we can, I will try!

Reply:
>So could you be more specific of what you would except?

Why not just add your 2 new plots (n_samples vs fit_time and fit_time vs score) to both pre-existing estimators?

That would be a 3 by 2 grid of plots and would avoid having this whole new code section

Reply:
OK, so here are the resulting plots with changes of ea0cc71:

![image](https://user-images.githubusercontent.com/16240134/58532814-f2f34080-81e6-11e9-9c0b-dd690ca6439a.png)

Reply:
If you look at the generated doc https://59951-843222-gh.circle-artifacts.com/0/doc/auto_examples/model_selection/plot_learning_curve.html

the plots are very small. Maybe try with 3 rows and 2 columns instead.
(you can build the doc locally following these guidelines https://scikit-learn.org/dev/developers/contributing.html#building-the-documentation)

Reply:
I am having trouble generating the doc. I am using a miniconda venv so I enter the following in the prompt command:
```
(py3) path\scikit-learn\doc>set EXAMPLES_PATTERN=plot_learning_curve.py
(py3) path\scikit-learn\doc>make html
```
But all the examples are then generated (regex var not taken in account) and this takes forever on my computer...
I also tried with `(py3) path\scikit-learn\doc>set EXAMPLES_PATTERN=*plot_learning_curve.py` but same result. How would you do?

Reply:
the doc says `EXAMPLES_PATTERN=your_regex_goes_here make html`, in one command

Reply:
This gave me unknown command error